whitepaper/chapters/reputation.tex

```diff
@@ -31,7 +31,7 @@ \subsection{Connection Management}
 This guarantees that over time each node will make connections to many different nodes in the network.
 
 \subsection{Weight Based Node Selection}
-\index{reputation!node!selection}
+\index{node!selection}
```

Comment:
why ~~reputation~~?

Reply:
for nicer index (flatten to at most two everywhere)